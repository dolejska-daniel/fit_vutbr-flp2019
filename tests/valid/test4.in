11,22,33,44,55,66,77,88
abc
11
66
11,a,33
11,b,22
22,a,44
22,b,66
33,b,44
44,c,55
55,a,55
77,b,66
88,c,77
