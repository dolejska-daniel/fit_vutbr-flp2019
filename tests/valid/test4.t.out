0,1,2,3,4,5
abc
0
4
0,a,1
0,b,2
1,b,3
2,a,3
2,b,4
3,c,5
5,a,5
